Orange/widgets/unsupervised/owmds.py

```diff
@@ -392,17 +395,32 @@ def on_partial_result(self, result: Result):
             if need_update:
                 self.graph.update_coordinates()
                 self.graph.update_density()
+                self.update_stress()
 
     def on_done(self, result: Result):
         assert isinstance(result.embedding, np.ndarray)
         assert len(result.embedding) == len(self.effective_matrix)
         self.embedding = result.embedding
         self.graph.update_coordinates()
         self.graph.update_density()
+        self.update_stress()
         self.run_button.setText("Start")
         self.step_button.setEnabled(True)
         self.commit.deferred()
 
+    def update_stress(self):
+        self.stress = self._compute_stress()
+        stress_val = "-" if self.stress is None else f"{self.stress:.3f}"
+        self.stress_label.setText(f"Kruskal Stress: {stress_val}")
+
+    def _compute_stress(self):
+        if self.embedding is None or self.effective_matrix is None:
+            return None
+        actual = scipy.spatial.distance.pdist(self.embedding)
+        actual = scipy.spatial.distance.squareform(actual)
+        return np.sqrt(np.sum((actual - self.effective_matrix) ** 2)
+                       / (np.sum(self.effective_matrix ** 2) or 1))
```

Comment:
It would probably be better to call `get_stress` and normalize the returned result.

Reply:
Done.